db.go

```diff
@@ -9,16 +9,21 @@ import (
 
 	"github.com/btcsuite/btcutil"
 	"github.com/btcsuite/btcwire"
+	"golang.org/x/crypto/ripemd160"
 )
 
 // Errors that the various database functions may return.
 var (
+	ErrAddrIndexDoesNotExist  = errors.New("Address index hasn't been built up yet.")
```

Comment:
Don't use capitals and punctuation in error strings.

golint should catch this but for some reason the travis build still succeeded.